Cows-HacknSlash/Assets/Scripts/PlayerController/CharController.cs

```diff
@@ -50,9 +50,12 @@ public class CharController : MonoBehaviour {
 	}
 
     private void OnTriggerEnter(Collider other) {
+        if (other.tag == "Fire_Ball_Enemy") {
```

Comment:
Please do not use `==` comparison on strings, rather use `other.tag.Equals("Fire_Ball_Enemy")`